static/fluent/pl/main.ftl

```diff
@@ -0,0 +1,524 @@
+
+## User Settings
+user-settings-input-name =
+  .placeholder = Imię
+user-settings-input-email =
+  .placeholder = Email
+user-settings-input-password =
+  .placeholder = Hasło
+user-settings-input-new-password =
+  .placeholder = Nowe Hasło (Opcjonalne)
+user-settings-input-confirm-new-password =
+  .placeholder = Potwierdź Nowe Hasło
+user-settings-input-confirm-password =
+  .placeholder = Potwierdź Hasło
+user-settings-password-mismatch = Hasła są niezgodne
+user-settings-save = Zapisz
+
+## Adapter Settings
+adapter-settings-no-adapters = Brak dostępnych adapterów.
+
+## Authorization Settings
+authorization-settings-no-authorizations = Brak utworzonych autoryzacji.
+
+## Experiment Settings
+experiment-settings-smart-assistant = Inteligentny Asystent
+experiment-settings-logs = Wykresy
+
+## Localization Settings
+localization-settings-language-region = Język i Region
+localization-settings-country = Kraj
+localization-settings-timezone = Strefa czasowa
+localization-settings-language = Język
+localization-settings-units = Jednostki
+localization-settings-units-temperature = Temperatura
+localization-settings-units-temperature-celsius = Temperatura Celsjusza (°C)
+localization-settings-units-temperature-fahrenheit = Temperatura Fahrenheita (°F)
+
+## Update Settings
+update-settings-update-now = Aktualizuj teraz
+update-available = Dostępna nowa wersja
+update-up-to-date = Twój system jest aktualny
+last-update = Ostatnia aktualizacja
+current-version = Obecna wersja
+failed = Niepowodzenie
+never = Nigdy
+in-progress = W trakcie
+restarting = Restartowanie
+
+## Developer Settings
+developer-settings-enable-ssh = Włącz SSH
+developer-settings-view-internal-logs = Wyświetl Dzienniki Wewnętrzne
+developer-settings-create-local-authorization = Utwórz autoryzację lokalną
+
+## Rules
+rules = Reguły
+rules-create-rule-hint = Nie utworzono reguł. Kliknij +, aby utworzyć regułę.
+rules-rule-name = Nazwa Reguły
+rules-customize-rule-name-icon =
+  .alt = Dostosuj Nazwę Reguły
+rules-rule-description = Opis Reguły
+rules-preview-button =
+  .alt = Podgląd
+rules-delete-icon =
+  .alt = Usuń
+rules-drag-hint = Przeciągnij tutaj urządzenia, aby rozpocząć tworzenie reguły
+rules-drag-input-hint = Dodaj urządzenie jako wejściowe
+rules-drag-output-hint = Dodaj urządzenie jako wyjściowe
+rules-scroll-left =
+  .alt = Przewiń w lewo
+rules-scroll-right =
+  .alt = Przewiń w prawo
+rules-delete-prompt = Upuść urządzenie tutaj, aby rozłączyć
+rules-delete-dialog = Czy na pewno chcesz trwale usunąć te reguły?
+rules-delete-cancel =
+  .value = Anuluj
+rules-delete-confirm =
+  .value = Usuń Reguły
+rule-invalid = Niepoprawna
+rule-delete-prompt = Czy na pewno chcesz trwale usunąć tę regułę?
+rule-delete-cancel-button =
+  .value = Anuluj
+rule-delete-confirm-button =
+  .value = Usuń Regułę
+rule-select-property = Wybierz Właściwość
+rule-not = Nie
+rule-event = Zdarzenie
+rule-action = Akcja
+rule-configure = Konfiguruj...
+rule-time-title = Godzina
```

Comment:
I think it would make sense to translate as "Czas" since you can set a specific time like "13:45", while "Godzina" suggests (at least to me) that you can set either 13:00 or 14:00 but nothing in between.

Reply:
Done.